src/_component/connect/connect.scss

```diff
@@ -0,0 +1,85 @@
+
+* {
+  box-sizing: border-box;
+}
+.connect {
+  @extend %align-items;
+}
+.connect-title {
+@extend %red-box;
+@extend %font;
+}
+
+.connect-box {
+  @extend %red-header;
+}
+
+.connect-avatar {
+  width: 100%;
+  border-radius: 5px;
+  border: 1px solid hsl(0, 7%, 74%);
+}
+
+.connect-avatar__wrapper {
+  border-radius:6px;
+  padding: 8px 8px 4px;
+  border: 1px solid hsl(0, 7%, 74%);
+  margin-bottom: 10px;
+  box-shadow: 0 5px 10px rgba(0, 0, 0, 0.25);
+}
+
+.hover-effect {
+  border-radius: 7px;
+  -webkit-transition: -webkit-transform .8s ease-in-out;
```

Comment:
in one place you have 0.25 and in one you are missing the 0, you got .8s
inconsitency